service/collector/cluster/conditions.go

```diff
@@ -0,0 +1,89 @@
+package cluster
+
+import (
+	"context"
+
+	"github.com/giantswarm/apiextensions/v2/pkg/label"
+	aeconditions "github.com/giantswarm/apiextensions/v3/pkg/conditions"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/prometheus/client_golang/prometheus"
+	capiv1alpha3 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	"sigs.k8s.io/cluster-api/util/conditions"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type Conditions struct {
+	ctrlClient client.Client
+	logger     micrologger.Logger
+}
+
+var (
+	clusterStatus = prometheus.NewDesc(
+		prometheus.BuildFQName(MetricsNamespace, "cluster", "status"),
+		"Latest cluster status conditions as provided by the Cluster CR status.",
+		[]string{
+			"cluster_id",
+			"release_version",
+			"status",
+		},
+		nil,
+	)
```

Comment:
btw. this seems very much like the metric we want to incorporate into cortex release info :stuck_out_tongue: